runtime/gc_glue_java/EnvironmentDelegate.hpp

```diff
@@ -197,6 +203,18 @@ class MM_EnvironmentDelegate
 			/* calculate this BEFORE we (potentially) destroy the object */
 			_gcEnv.movedObjectHashCodeCache.originalHashCode = computeObjectAddressToHash((J9JavaVM *)_extensions->getOmrVM()->_language_vm, objectPtr);
 		}
+
+#if defined(J9VM_ENV_DATA64)
+		if (objectModel->isIndexable(objectPtr)) {
+			GC_ArrayObjectModel *indexableObjectModel = &_extensions->indexableObjectModel;
+			/* check isVirtualLargeObjectHeapEnabled instead of isIndexableDataAddrPresent to avoid time waste for offheap disabled. */
```

Comment:
this refers to some historic code (isIndexableDataAddrPresent used be here), so this can be removed now